src/prometheus_sup.erl

```diff
@@ -17,10 +17,10 @@
 
 -spec start_link() -> supervisor:startlink_ret().
 start_link() ->
-    supervisor:start_link({local, ?MODULE}, ?MODULE, []).
+    supervisor:start_link({local, ?MODULE}, ?MODULE, noargs).
 
--spec init([]) -> {ok, {supervisor:sup_flags(), [supervisor:child_spec()]}}.
-init([]) ->
+-spec init(noargs) -> {ok, {supervisor:sup_flags(), [supervisor:child_spec()]}}.
+init(noargs) ->
```

Comment:
This change is pointless.

Reply:
I mean, if we all sneak in our prefered styles without care like this, the project is quickly going to look weird. Please remove it, it has nothing to do with this PR.

Reply:
Fair enough, undone 👌🏽